roqoqo-qryd/src/api_backend.rs

```diff
@@ -773,6 +842,28 @@ impl APIBackend {
     pub fn set_dev(&mut self, dev: bool) {
         self.dev = dev;
     }
+
+    /// Transforms a PragmaRepeatedMeasurement operation into a set of
+    /// MeasureQubit operations followed by a PragmaSetNumberOfMeasurements.
+    ///
+    fn _transform_pragma_repeated_measurements(
+        &self,
+        operation: PragmaRepeatedMeasurement,
+    ) -> Circuit {
+        let mut equivalent_circuit = Circuit::new();
+        let num_qubits = match &self.device {
+            QRydAPIDevice::QrydEmuSquareDevice(sq) => sq.number_qubits(),
+            QRydAPIDevice::QrydEmuTriangularDevice(tr) => tr.number_qubits(),
+        };
```

Comment:
That should work with calling number_qubits() on device directly as long as QRydAPIDevice implements the roqoqo Device trait. Also would make it immediately compatible with the new TweezerDevice